samples/wifi/scan/Kconfig

```diff
@@ -15,4 +15,10 @@ config WIFI_MAC_ADDRESS
 		WiFi MAC address to be used by the sample. If not set, the sample will
 		use the default MAC address (locally administrative). This is only used
 		when the nRF7002 OTP is not programmed with a MAC address.
+
+config WIFI_SCAN_DEFAULT
```

Comment:
This doesn't make sense, there is no need for a configuration to say that use defaults :). I guess you need to add `FORCE_PASSIVE_SCAN` and make `active` as default.
[Edit: Looking at next commit, this should be a `choice` of  scan types with `active` as default]